Makefile.am

```diff
@@ -64,7 +66,9 @@ AM_CPPFLAGS=		-I$(srcdir)/libcperciva/alg		\
 			-I$(srcdir)/lib/scryptenc 		\
 			-I$(srcdir)/lib/util			\
 			-DCPUSUPPORT_CONFIG_FILE=\"cpusupport-config.h\" \
-			-D_POSIX_C_SOURCE=200809L
+			-D_POSIX_C_SOURCE=200809L		\
+			-D_XOPEN_SOURCE=700			\
```

Comment:
Technically we don't need `XOPEN_SOURCE` in here, but our other software projects use it, so I added it in the interest of simplicity.